docs/modules/ROOT/partials/getting-started/con-registry-serdes-json.adoc

```diff
@@ -4,39 +4,41 @@
 [id='registry-serdes-types-json-{context}']
 = Configure JSON Schema SerDe with {registry}
 
-{registry} provides Kafka client serializer and deserializer classes for JSON Schema to make using JSON Schema as easy as possible:
+[role="_abstract"]
+{registry} provides the following Kafka client serializer and deserializer classes for JSON Schema:
 
-* `io.apicurio.registry.utils.serde.JsonSchemaKafkaSerializer`
-* `io.apicurio.registry.utils.serde.JsonSchemaKafkaDeserializer`
+* `io.apicurio.registry.serde.jsonschema.JsonSchemaKafkaSerializer`
+* `io.apicurio.registry.serde.jsonschema.JsonSchemaKafkaDeserializer`
 
-Unlike Apache Avro, JSON Schema is not actually a serialization technology - it is instead a validation
-technology. As a result, configuration options for JSON Schema are quite different. For example, there is no
-encoding option, because data is always encoded as JSON.
+Unlike Apache Avro, JSON Schema is not a serialization technology, and is instead a validation technology. As a result, configuration options for JSON Schema are quite different. For example, there is no encoding option, because data is always encoded as JSON.
 
 .Configure the JSON Schema serializer
 
-You can configure the JSON Schema serializer class in the following ways:
+You can configure the JSON Schema serializer class as follows:
 
-* {registry} location as a URL
-* Artifact ID strategy 
-* Global ID strategy
+* {registry} URL
+* Artifact resolver strategy 
+* Global ID strategy 
 * Validation enabled/disabled
 
 The only non-standard configuration property is whether JSON Schema validation is enabled or
-disabled.  The validation feature is disabled by default but can be enabled by setting
-`apicurio.registry.serdes.json-schema.validation-enabled` to `"true"`. For example:
+disabled.  The validation feature is disabled by default but you can enable this by setting
+`apicurio.registry.serde.validation-enabled` to `"true"`. For example:
 ----
-props.putIfAbsent(JsonSchemaSerDeConstants.REGISTRY_JSON_SCHEMA_VALIDATION_ENABLED, "true")`
+props.putIfAbsent(SerdeConfig.VALIDATION_ENABLED, Boolean.TRUE)`
 ----
 
 .Configure the JSON Schema deserializer
 
 You can configure the JSON Schema deserializer class in the following ways:
 
-* {registry} location as a URL
+* {registry} URL
```

Comment:
for the json deserializer there are new configuration options, check `io.apicurio.registry.serde.jsonschema.JsonSchemaKafkaDeserializerConfig`